stratum/hal/config/x86-64-netberg-aurora-710-r0/port_map.json

```diff
@@ -0,0 +1,3516 @@
+{
+  "_comment": "Configuration file for bspless support on x86-64-netberg-aurora-750-r0",
```

Comment:
`x86-64-netberg-aurora-710-r0` ?

Reply:
Maybe this is a wrong port map file for Aurora 710 since it has more than 32 ports(connectors).
Can you check again?